tools/ja-rule/ja-rule.cpp

```diff
@@ -407,8 +709,20 @@ class InputHandler {
 int main(int argc, char **argv) {
   ola::AppInit(&argc, argv, "[ options ]", "Ja Rule Admin Tool");
 
+  auto_ptr<UID> target_uid(UID::FromString(FLAGS_target_uid));
+  if (!target_uid.get()) {
```

Comment:
This should probably check or at least warn that the UID isn't a broadcast one, or do you want to be able to test odd stuff?